test/run_test.py

```diff
@@ -1161,36 +1167,66 @@ def query_changed_test_files() -> List[str]:
     lines = [line.strip() for line in lines]
     return lines
 
```

Comment:
missing an extra blank line here
```suggestion


```

Reply:
I actually dont know the rules for extra blank line. is it possible to enforce it via linter?

Reply:
yeah we could use something like [Black](https://black.readthedocs.io/en/stable/) but that would annoy people so we don't

Reply:
but it's conventional to put two blank lines around top-level function or class definitions